src/daemon/screenlock.vala

```diff
@@ -68,16 +90,19 @@ namespace Budgie {
 
 				/**
 				* Try in order, and load the first one that exists:
+				* - ~/.config/budgie-desktop/[gtklock.ini | gtklock.css]
 				* - /etc/budgie-desktop/[gtklock.ini | gtklock.css]
 				* - /usr/share/budgie-desktop/[gtklock.ini | gtklock.css]
 				*/
-				string[] system_configs = {
+				string[] lock_configs = {
+					"file://" + Environment.get_user_config_dir() + "/budgie-desktop/gtklock.ini",
 					@"file://$(Budgie.CONFDIR)/budgie-desktop/gtklock.ini",
 					@"file://$(Budgie.DATADIR)/budgie-desktop/gtklock.ini"
 				};
 
-				foreach (string? filepath in system_configs) {
+				foreach (string? filepath in lock_configs) {
 					File file = File.new_for_uri(filepath);
+					warning(filepath);
```

Comment:
Remove warning. Guessing it was there when you were debugging the addition of the gtklock.css from user config dir.

```suggestion
```